app/models/role.rb

```diff
@@ -212,4 +217,9 @@ def find_filter(resource_type, current_filters, search)
       self.filters.build(:search => search)
     end
   end
+
+  def invalid_tax_error(tax)
+    msg = "cannot be cloned. Some #{tax} are invalid. Please fix their associations and try again."
+    errors.add(:base, _(msg))
```

Comment:
I'm afraid gettext won't be able to correctly find the string this way. You need to wrap the string definition with `N_(...)` and extract `#{tax}` into string substitution or define it twice. The latter is probably better because we translate words "organization" and "location" too.